test/evergreen.yml

```diff
@@ -558,6 +558,15 @@ variables:
         vars:
           format_test_script_args: -e "SEGFAULT_SIGNALS=all" -b "catchsegv ./t" -t 360
 
+  - &format-abort-stress-test
+    exec_timeout_secs: 3600
```

Comment:
Can you change the exec_timeout_secs to 2500? Because let's say we are running test/format for 30 minutes(-t 30 -> 1800 seconds) and ~10 minutes for compile task.

Reply:
Sure

Reply:
998d609d2be9a03f408952b738ee5cc66205bee2